src/code/z_parameter.c

```diff
@@ -1,4 +1,264 @@
+    { SCENE_MAP_SHOP, 0x10, 0x30, 0xC1 },
+    { SCENE_F40, 0x00, 0x00, 0x00 },
+    { SCENE_F41, 0x00, 0x00, 0x00 },
+    { SCENE_10YUKIYAMANOMURA2, 0x00, 0x00, 0x00 },
+    { SCENE_14YUKIDAMANOMITI, 0x00, 0x00, 0x00 },
+    { SCENE_12HAKUGINMAE, 0x00, 0x00, 0x00 },
+    { SCENE_17SETUGEN, 0x00, 0x00, 0x00 },
+    { SCENE_17SETUGEN2, 0x00, 0x00, 0x00 },
+    { SCENE_SEA_BS, 0x00, 0x00, 0x00 },
+    { SCENE_RANDOM, 0x00, 0x00, 0x00 },
+    { SCENE_YADOYA, 0x10, 0x30, 0xC1 },
+    { SCENE_KONPEKI_ENT, 0x00, 0x00, 0x00 },
+    { SCENE_INSIDETOWER, 0x00, 0xFF, 0xC0 },
+    { SCENE_26SARUNOMORI, 0x00, 0x30, 0x00 },
+    { SCENE_LOST_WOODS, 0x00, 0x00, 0x00 },
+    { SCENE_LAST_LINK, 0x00, 0x3F, 0x00 },
+    { SCENE_SOUGEN, 0x00, 0x3F, 0x00 },
+    { SCENE_BOMYA, 0x10, 0x30, 0xC1 },
+    { SCENE_KYOJINNOMA, 0x00, 0x00, 0x00 },
+    { SCENE_KOEPONARACE, 0x00, 0x30, 0x00 },
+    { SCENE_GORONRACE, 0x00, 0x00, 0x00 },
+    { SCENE_TOWN, 0x00, 0x00, 0x00 },
+    { SCENE_ICHIBA, 0x00, 0x00, 0x00 },
+    { SCENE_BACKTOWN, 0x00, 0x00, 0x00 },
+    { SCENE_CLOCKTOWER, 0x00, 0x00, 0x00 },
+    { SCENE_ALLEY, 0x00, 0x00, 0x00 },
+};
+
+s16 D_801BF884 = 0;     // pictoBox related
+s16 D_801BF888 = false; // pictoBox related
+
+s16 sHBAScoreTier = 0;
+
+u16 sMinigameScoreDigits[] = { 0, 0, 0, 0 };
+
+u16 sCUpInvisible = 0;
+u16 sCUpTimer = 0;
+
+s16 sMagicBarOutlinePrimRed = 255;
+s16 sMagicBarOutlinePrimGreen = 255;
+s16 sMagicBarOutlinePrimBlue = 255;
+
+s16 D_801BF8AC = 2; // sMagicBorderRatio
+
+s16 D_801BF8B0 = 1;
+
+s16 sExtraItemBases[] = {
+    ITEM_STICK,   ITEM_STICK, ITEM_NUT,   ITEM_NUT, ITEM_BOMB,    ITEM_BOMB,    ITEM_BOMB,
+    ITEM_BOMB,    ITEM_BOW,   ITEM_BOW,   ITEM_BOW, ITEM_BOMBCHU, ITEM_BOMBCHU, ITEM_BOMBCHU,
+    ITEM_BOMBCHU, ITEM_STICK, ITEM_STICK, ITEM_NUT, ITEM_NUT,
+};
+
+s16 D_801BF8DC = 0;
+
+s16 D_801BF8E0 = 0;
+
+s16 D_801BF8E4 = 0;
+
+OSTime D_801BF8E8 = 0;
+OSTime D_801BF8F0 = 0;
+OSTime D_801BF8F8[] = {
+    0, 0, 0, 0, 0, 0, 0,
+};
+OSTime D_801BF930[] = {
+    0, 0, 0, 0, 0, 0, 0,
+};
+
+u8 D_801BF968 = false;
+u8 D_801BF96C = false;
+
+s16 D_801BF970 = 99;
+
+s16 D_801BF974 = 0;
+
+s16 D_801BF978 = 10;
+
+s16 D_801BF97C = 255;
+
+f32 D_801BF980 = 1.0f;
+
+s32 D_801BF984 = 0;
+
+// Display List
+Gfx D_801BF988[] = {
+    gsDPPipeSync(),
+    gsSPClearGeometryMode(G_ZBUFFER | G_SHADE | G_CULL_BOTH | G_FOG | G_LIGHTING | G_TEXTURE_GEN |
+                          G_TEXTURE_GEN_LINEAR | G_LOD | G_SHADING_SMOOTH),
+    gsDPSetOtherMode(G_AD_DISABLE | G_CD_MAGICSQ | G_CK_NONE | G_TC_FILT | G_TF_BILERP | G_TT_NONE | G_TL_TILE |
+                         G_TD_CLAMP | G_TP_NONE | G_CYC_1CYCLE | G_PM_1PRIMITIVE,
+                     G_AC_NONE | G_ZS_PIXEL | G_RM_CLD_SURF | G_RM_CLD_SURF2),
+    gsDPSetCombineMode(G_CC_PRIMITIVE, G_CC_PRIMITIVE),
+    gsSPEndDisplayList(),
+};
+
+s16 D_801BF9B0 = 0;
+
+f32 D_801BF9B4[] = {
+    100.0f,
+    109.0f,
+};
+
+s16 D_801BF9BC[] = {
+    0x226,
+    0x2A8,
+    0x2A8,
+    0x2A8,
+};
```

Comment:
Personally I prefer these shorter arrays to be formatted without a trailing comma, since they take up so much vertical space. However it is up to you.